deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-service-core.ts

```diff
@@ -86,19 +76,7 @@ export class MigrationServiceCore extends Stack {
             props.volumes.forEach(vol => serviceTaskDef.addVolume(vol))
         }
 
-        let serviceImage
-        if (props.dockerDirectoryPath) {
-            serviceImage = ContainerImage.fromDockerImageAsset(new DockerImageAsset(this, "ServiceImage", {
-                directory: props.dockerDirectoryPath,
-                buildArgs: props.dockerBuildArgs,
-                // File path relative to above directory path
-                file: props.dockerFilePath
-            }))
-        }
-        else {
-            // @ts-ignore
-            serviceImage = ContainerImage.fromRegistry(props.dockerImageRegistryName)
-        }
+        const serviceImage = makeDockerImageAsset(this, props.serviceName, props.dockerImageRegistryName)
```

Comment:
rename 'dockerImageRegistryName' - upon first glance, I thought that it meant the name of your registry, not the name within the registry

Reply:
Renamed